runners/console/index.ts

```diff
@@ -52,6 +52,14 @@ export class Console extends Runner {
         return result;
     }
 
+
+    runRestoreDevonfwIde(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;    
+        result = this.runInstallDevonfwIde(step, command);
```

Comment:
Are those lines necessary? Or is 
let result = this.runInstallDevonfwIde(step, command) 
also possible?

Reply:
These are not necessary. We can simply write return this.runInstallDevonfwIde(step, command)